src/bin/gpfdist/regress/input/gpfdist2.source

```diff
@@ -23,6 +23,17 @@ CREATE TABLE REG_REGION (R_REGIONKEY INT, R_NAME CHAR(25), R_COMMENT VARCHAR(152
 -- select * from check_ps;
 -- select * from check_env;
 
+-- Helper to remove the output files
+DROP EXTERNAL WEB TABLE IF EXISTS clean_out_files;
+CREATE EXTERNAL WEB TABLE clean_out_files (x text)
+EXECUTE E'(rm -f @abs_srcdir@/data/gpfdist2/lineitem.tbl.out.zst @abs_srcdir@/data/gpfdist2/test_quote.csv)'
+on SEGMENT 0
+FORMAT 'text';
+
+SELECT * FROM clean_out_files;
```

Comment:
Why have you done this before test? What about move it to the and of the test?

Reply:
I can see 2 pros of cleaning before:
- we are tolerant to a state left from previous runs 
- after the run there are artifacts left, so we can analyze them postmortem
Also, in the other file in this PR there are some artifacts which also cleaned before the test.